yum/artifact-registry.py

```diff
@@ -27,6 +27,11 @@ def prereposetup_hook(conduit):
   if not token:
     return
   for repo in conduit.getRepos().listEnabled():
+    # Check if the custom 'ar_auth' option is set in the repository's config.
+    if repo.cfg.has_option(repo.id, 'ar_auth') and repo.cfg.getboolean(repo.id, 'ar_auth'):
```

Comment:
@ericdand Let me know if you'd rather me use a different property than `ar_auth`

Reply:
I might suggest "artifact_registry_oauth" instead. That will be much clearer when a sysadmin opens the config file without context; it states both the full name of the service, and what kind of auth is used. :)

Reply:
Another option would be `google_artifact_registry_oauth`, which might be even clearer. Up to you if you really need "Google" on there, though. It's a balance of clarity and conciseness (concision? idk, I'm a software engineer, not an English major).